projects/archiver/src/tasks/notification/helpers/device-notifications-helpers.ts

```diff
@@ -57,17 +58,13 @@ export const prepareScheduleDeviceNotificationRequest = (
 ): { reqEndpoint: RequestInfo; reqBody: RequestInit } => {
     const { domain, apiKey } = cfg
 
-    const { key, name, issueDate } = issueData
+    const { key, name, issueDate, edition } = issueData
+    const topicName = getEditionCountryBy(edition)
```

Comment:
this would publish training and daily to the same topic?

Reply:
yeah it looks confusing, i agree
For now we will send the notifications only for Daily Editions, so i can keep the hardcoded option with comment

Reply:
i took that out :)